roles/openshift_logging/tasks/main.yaml

```diff
@@ -12,6 +12,14 @@
 
 - debug: msg="Created temp dir {{mktemp.stdout}}"
 
+- name: Create local temp directory for doing work in
```

Comment:
My only concern is that we are now creating a temp dir on the control host but then we are then doing a remote command to copy the kubeconfig to the created temp dir. This would work if our remote and control host are the same, but I'm not sure it will when they are different...